### PR TITLE
New UI, Navbar, 2 new Charts, and fixes.

### PR DESCRIPTION
**Pull request by spmcbride1201**
UI:
- Add oncanvas sidebar with offcanvas sidebar for mobile
- Expand container to use full screen real estate

Charts:
- Add JVM Heap Chart: Shows the utilization patterns of heap memory
- Add Data Throughput Chart: Shows data transfer rates up and down in Kbps
- Fixed an issue in the HTTP Connections chart where I was excluding zero values via a poorly-written filter that was checking for falsey values.

Components:
- Add dummy Admin component for future use

### CODE REVIEW

File: src/components/DataTxLineChart.js [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+export default function DataTxLineChart({ metricsArr }) {
+
+  // to derive the "per second" value, I need the refresh interval to live in the Redux store
+  const INTERVAL_HACK = 5;
+  // Map the metrics array and generate array of objects with an array of objects
+  // with timestamps and the net change since the last metrics snapshot. The initial snapshot
+	// is considered a net change of zero
+  const deltas = metricsArr.map(function (metrics, index) {
+    const date = new Date(metrics.date);
+    const prettyDate = dateFormat(date, "h:MMtt");
```
Comment (tilleryd) [inline src/components/DataTxLineChart.js]: Nice, I need to try `dateFormat`.

File: src/components/HTTPGrid.js [hunk 2]
```diff
@@ -1,62 +1,32 @@
...
       <div>
-        <h1 className="uk-heading-line uk-text-center"><span>HTTP</span></h1>
         <div
-          className="uk-grid-match uk-grid-small uk-text-center"
-          data-uk-grid
+          className="uk-grid-match uk-text-center"
+          data-uk-grid 
+					data-uk-grid-small
+					data-uk-height-match="row: false"
+					data-uk-sortable
```
Comment (tilleryd) [inline src/components/HTTPGrid.js]: Oops some indentation issues.

File: src/components/JVMHeapLineChart.js [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+  });
+
+  return (
+    <div className="uk-card uk-card-default uk-card-body">
+      <h3 className="uk-card-title">JVM Heap</h3>
+      <ResponsiveContainer
+        height="80%"
+        width="90%"
+      >
+        <LineChart data={httpConnections}>
```
Comment (tilleryd) [inline src/components/JVMHeapLineChart.js]: I'm liking these `Recharts` components.

Comment (tilleryd) [inline src/components/JVMHeapLineChart.js]: It looks like so little effort to get a D3 based chart working in React using it.

File: src/style/index.css [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+	color: #999
+}
+
+.hamburger {
+	color: #ddd
+}
+
+.hamburger:hover {
+	color: #fff
+}
```
Comment (tilleryd) [inline src/style/index.css]: Overly indented properties?

File: src/components/HTTPGrid.js (hunk 2 above)
Comment (spmcbride1201) [inline src/components/HTTPGrid.js]: Oh man, there is some weird misconfiguration that does weird spaces to tabs stuff that isn't immediately apparent. I'm finally frustrated enough to try to figure in out.

File: src/style/index.css (hunk 4 above)
Comment (spmcbride1201) [inline src/style/index.css]: Eek. Same weird spaces to tabs issues with my autoformatter... My b.

File: src/components/JVMHeapLineChart.js (hunk 3 above)
Comment (spmcbride1201) [inline src/components/JVMHeapLineChart.js]: Yeah, I think they're the slickest React component black boxes out there for D3. I also like Sparkcharts for really simple 'at a glance' style line charts, which are even easier to use. I figure we can flush out stylized recharts and be really selective about custom D3 visualizations (if we end up needing them at all).

**Pull request merged**